Automattic/jetpack
# Social: Add connection modal to the editor

Fixes https://github.com/Automattic/jetpack-reach/issues/313

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/313
## Does this pull request change what data or activity we track or use?


No
## Testing instructions:






Test with Jetpack + Social and with Social only

* Go to the new post page
* Clicking the + button should bring up the new connection modal
* ~~If you have #37295 or it's merged since, you can fully add the connection~~
* Add a new connection
* After connecting it should appear in the list in the editor
